src/v/config/convert.h

```diff
@@ -475,37 +474,6 @@ struct convert<model::cloud_storage_chunk_eviction_strategy> {
     }
 };
 
-template<>
-struct convert<pandaproxy::schema_registry::subject_name_strategy> {
```

Comment:
@BenPope all indications are that this was dead code, and wasn't triggered by any sort of external input. still, i was surprised.

Reply:
> @BenPope all indications are that this was dead code, and wasn't triggered by any sort of external input. still, i was surprised.

Good catch. At [some point during the implementation](https://github.com/redpanda-data/redpanda/pull/10650#issuecomment-1563757485) it was decided that it was unnecessary to be able to change the global default.